src/test/resources/s3-valid-inbound-cases.xml

```diff
@@ -66,6 +66,22 @@
 		<integration:poller fixed-rate="1000"/>
 	</int-aws:s3-inbound-channel-adapter>
 
+	<int-aws:s3-inbound-channel-adapter
+			id="validInboundWithProxy"
+			bucket="TestBucket"
+			propertiesFile="testawscredentials.properties"
+			temporary-suffix=".temp"
+			local-directory="${foo.dir:/}"
+			remote-directory="remote"
+			accept-sub-folders="true"
+			max-objects-per-batch="100"
+			file-name-regex="[A-Za-z0-9]+\\.txt"
+			channel="inbound"
+			auto-startup="false"
+            proxyHost="localhost"
+            proxyPort="80">
```

Comment:
Thanks for your feedback, as per your comment are you suggesting we will have custom S3Operations with Client Configuration? If so only modification would be to the Abstract Class to have a another Constructor to accept the Client Configuration. Please correct my understanding.